build.gradle.kts

```diff
@@ -87,6 +87,14 @@ subprojects {
 
 			// Hystrix
 			api(group = "com.netflix.hystrix", name = "hystrix-core", version = "1.5.18")
+
+			// rich version declarations
+			add("api", "com.fasterxml.jackson.core:jackson-databind") {
```

Comment:
do we also apply to `jackson-core`, `jackson-annotations`, and `jackson-datatype-jsr310` to be safe?